.gitlab/e2e.yml

```diff
@@ -260,13 +261,25 @@ new-e2e-cws-dev:
   variables:
     TARGETS: ./tests/cws
     TEAM: csm-threats-agent
+  parallel:
+    matrix:
+      - OVERRIDE_CI_JOB_NAME: new-e2e-cws-agent-dev
+        EXTRA_PARAMS: --run TestAgentSuite
+      - OVERRIDE_CI_JOB_NAME: new-e2e-cws-fargate-dev
+        EXTRA_PARAMS: -c cwsinstrumentation:fullImagePath=669783387624.dkr.ecr.us-east-1.amazonaws.com/cws-instrumentation:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA} --run TestECSFargate
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As an alternative you could take `cwsinstrumentation:fullImagePath=669783387624.dkr.ecr.us-east-1.amazonaws.com/cws-instrumentation:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}` as a test flag or variable, set it for all tests, use it only in your `TestECSFargate`. Should normalise the test name

Something like

```
variables: 
      CWS_WORKLOAD_IMAGE_PATH:.....
```

And then in your fargate test

```
configMap[cwsinstrumentation] = os.Getenv("CWS_WORKLOAD_IMAGE_PATH")
```

Reply:
This seems to do the trick. I updated the PR to configure the cws-instrumentation image through an environment variable instead of using an extra `new-e2e-tests.run` command line parameter. Thanks for the suggestion!